wonjong/app.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const logger = require("morgan");
+const dotenv = require("dotenv");
+const { DataSource } = require("typeorm");
+const { json } = require("express");
+dotenv.config();
+app.use(logger("combined"));
+require("dotenv").config();
```

Comment:
- 불필요하게 작성된 코드 삭제해 주세요.(아래 예시코드 확인)

```suggestion
const cors = require("cors");
const app = express();
const logger = require("morgan");
const dotenv = require("dotenv");
const { DataSource } = require("typeorm");

dotenv.config();

app.use(logger("combined"));

```